packages/api/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "@tsconfig/recommended/tsconfig.json",
+  "include": ["src"],
+  "compilerOptions": {
+    "outDir": "dist",
+    "allowJs": true
```

Comment:
Temporary during the migration. Once all JS files are change to TS this can be removed.